cadcutils/dev_requirements.txt

```diff
@@ -0,0 +1,8 @@
+numpy==1.12.0b1
+astropy==1.2.1
+cadcutils==1.0
```

Comment:
Didn't see it in the vofs repo, removed it from the cadcdata/dev_requirements.txt though.